packages/cli/lib/cli.ts

```diff
@@ -225,6 +225,26 @@ NANGO_DEPLOY_AUTO_CONFIRM=false # Default value`
 
     await generate(debug, inParentDirectory);
 
+    // Install zod validation library
```

Comment:
What is this doing? Installing zod to the customers project?

Reply:
yes installing zod in their integration folder

Reply:
I'm not sure if this is a good idea, because 
1. some customers run the cli globally so there is no package.json
2. we're forcing customers to install something they may not use or want

Instead I think if someone wants to use zod they can be explicit and install it to use with their dry run

Reply:
1. It installs zod locally inside the integrations folder so the compiling step can use the package. 
zod is not a dependency of the CLI, it is a dependency of the scripts.

2. I was thinking the same initially but @bastienbeurier prefers we fully abstract its installation